jdaviz/core/loaders/importers/spectrum2d_as_list/spectrum2d_as_list.py

```diff
@@ -0,0 +1,57 @@
+from specutils import Spectrum1D, SpectrumList
+
+from jdaviz.core.registries import loader_importer_registry
+from jdaviz.core.loaders.importers import BaseImporterToDataCollection
+
+
+@loader_importer_registry('1D Spectrum List')
+class Spectrum2DAsListImporter(BaseImporterToDataCollection):
+    template_file = __file__, "spectrum2d_as_list.vue"
+
+    @property
+    def is_valid(self):
+        if self.app.config != 'specviz':
+            # NOTE: temporary during deconfig process
+            return False
+        # TODO: should this be split into two loaders?
+        # should a loader take a single input type, output a single output type,
+        # or just have a consistent data_label and viewer?
+        return (isinstance(self.input, SpectrumList) or (isinstance(self.input, Spectrum1D)
+                and self.input.flux.ndim == 2))
+
+    @property
+    def output(self):
+        if not self.is_valid:  # pragma: nocover
+            return None
+        if isinstance(self.input, SpectrumList):
+            return self.input
+        elif isinstance(self.input, Spectrum1D):
+            def this_row(field, i):
+                if field is None:
+                    return None
+                return field[i, :]
```

Comment:
Are we sure there won't be a case where we need to access `[:, i]` instead? Is this affected by specutils 2.0 work?  Is such access not implemented upstream in specutils ? Might be questions for @rosteen .

Reply:
I'm not sure, I just copied the minimal logic to get this running as possible from the existing parsers.  The idea here is for an example of multiple applicable importers - we'll have to build out and test all the actual importing logic later.